packages/generate-html/test/generate-html.spec.ts

```diff
@@ -1,251 +1,203 @@
-      schemaExtraTranslationsPath: `${__dirname}/../../../fixtures/CoA/v1.0.0/extra_translations.json`,
-      schemaInterface: readFileSync(`${__dirname}/../../../fixtures/CoA/v1.0.0/certificate.ts`, 'utf-8'),
-      expectedHtmlFromHbs: readFileSync(`${__dirname}/../../../fixtures/CoA/v1.0.0/template_hbs.html`, 'utf-8'),
-      expectedHtmlFromMjml: '',
-      partialsMap: {
-        company: 'https://schemas.s1seven.dev/schema-definitions/v0.0.5/company/company.hbs',
-      },
-    },
-    {
-      type: SupportedSchemas.COA,
-      version: 'v1.1.0',
-      certificatePath: `${__dirname}/../../../fixtures/CoA/v1.1.0/valid_cert.json`,
-      schemaTranslationsPath: `${__dirname}/../../../fixtures/CoA/v1.1.0/translations.json`,
-      schemaExtraTranslationsPath: `${__dirname}/../../../fixtures/CoA/v1.1.0/extra_translations.json`,
-      schemaInterface: readFileSync(`${__dirname}/../../../fixtures/CoA/v1.1.0/certificate.ts`, 'utf-8'),
-      localTemplatePath: `${__dirname}/../../../fixtures/CoA/v1.1.0/template.hbs`,
-      expectedHtmlFromHbs: readFileSync(`${__dirname}/../../../fixtures/CoA/v1.1.0/template_hbs.html`, 'utf-8'),
-      expectedHtmlFromMjml: '',
-      partialsMap: {
-        company: 'https://schemas.s1seven.dev/schema-definitions/v0.0.7/company/company.hbs',
-      },
-    },
-  ];
-
-  const htmlDifferOptions = {
-    ignoreAttributes: ['src'],
-    ignoreWhitespaces: true,
-    ignoreComments: true,
-    ignoreEndTags: true,
-    ignoreDuplicateAttributes: false,
+/*
+  When adding a new fixture version, add a new test suite to the testMap below.
+  Add new unreleased versions both to versions and unreleasedVersions. 
+  Remove from unreleasedVersions upon release.
+  To add more than one fixture per version, use the following naming format:
+  valid_cert_<number>.json
+  valid_cert_<number>.pdf
+  valid_cert_<number>.html
+  */
+
+const certificateTestMap = [
+  {
+    type: SupportedSchemas.EN10168,
+    versions: ['v0.1.0', 'v0.2.0', 'v0.3.0', 'v0.4.0', 'v0.4.1'],
+    unreleasedVersions: [],
+  },
+  {
+    type: SupportedSchemas.COA,
+    versions: ['v0.0.4', 'v0.1.0', 'v0.2.0', 'v1.0.0', 'v1.1.0'],
+    unreleasedVersions: [],
+  },
+];
+
+function generatePaths(validCertName: string, dirPath: string) {
+  const { name } = parse(validCertName);
+  const certificatePath = `${dirPath}/${name}.json`;
+  const schemaTranslationsPath = `${dirPath}/translations.json`;
+  const partialsMapPath = `${dirPath}/partials-map.json`;
+  const expectedHtmlFromHbs = readFileSync(`${dirPath}/${name}.html`, 'utf-8');
+  const schemaExtraTranslationsPath = existsSync(`${dirPath}/extra_translations.json`)
+    ? `${dirPath}/extra_translations.json`
+    : undefined;
+
+  let partialsMap: Record<string, string>;
+  try {
+    partialsMap = JSON.parse(readFileSync(partialsMapPath, 'utf-8'));
+  } catch (e) {
+    // partialsMap is not present
+    partialsMap = undefined;
```

Comment:
This is not strictly necessary, I left it here to make it obvious what is happening.